x-pack/plugin/sql/qa/src/main/resources/docs/docs.csv-spec

```diff
@@ -14,7 +14,7 @@ describeTable
 // tag::describeTable
 DESCRIBE emp;
 
-       column       |     type      |    mapping    
+       column       |     type      |    mapping
```

Comment:
Please revert all the formatting changes to untouched lines, it can cause confusion when looking at git history.